conf/application.conf

```diff
@@ -125,7 +125,7 @@ features {
   userResearchBanner.enabled = true
   allowDirectDebits.enabled = true
   staticDate.enabled = true
-  vatCertificate.enabled = true
+  paymentsAndRepaymentsEnabled.enabled = true
```

Comment:
You have this differently in the two app-config PRs. In those it's down as `paymentsAndRepayments.enabled`